package.json

```diff
@@ -11,19 +11,20 @@
   "bugs": "https://github.com/ain/smartbanner.js/issues",
   "dependencies": {},
   "devDependencies": {
+    "add-banner": "ain/add-banner#121b56d",
+    "autoprefixer": "^6.4.0",
     "babel-preset-es2015": "^6.9.0",
     "babelify": "^7.3.0",
+    "browserify": "^14.3.0",
```

Comment:
`browserify` is not required. `babelify` already has it. We should not have dependency change at all.